app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+# Users controller to manage show and update actions.
+class UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:update]
+  respond_to :json
+
+  def show
+    user = User.find(params[:id])
+    render json: user
+  end
+
+  def update
+    user = User.find(params[:id])
```

Comment:
you shouldn't query the user by id, the user that it going to be updated is the one currently sign in. You should be using current_user instead.